### ISSUE TITLE
login with organisation account on objecttypes reg admin generates wrong user name in backend

### ISSUE THREAD

**Issue by JurjenBert** [maykinmedia/objects-api#297]
![afbeelding](https://user-images.githubusercontent.com/40294293/170024078-d65daade-ff70-4611-9e8f-b5d1dde69dc9.png)
 
![afbeelding](https://user-images.githubusercontent.com/40294293/170024125-a9020f05-9ba6-498a-aa3f-c8f36bb220dd.png)

**Comment by sergei-maertens** [comment]
This is a configuration aspect in the OIDC configuration as far as I know. The OIDC protocol prescribes that unique identifiers have to be used, which is done here. If your organization can guarantee that another claim is also unique, it should be possible to specify this claim as username source instead, I think.

**Comment by damm89** [comment]
No, in mozilla_django_oidc_db version 0.6.0 you can't configure the username claim (stupidly enough?). It is always the "sub" key. Solution: bump objecttypes to latest version which has mozilla_django_oidc_db version 0.7.2 which allows for a custom username claim.
